develop/parachain-devs/parachain-launch/chain-spec.md

```diff
@@ -0,0 +1,113 @@
+---
+title: Chain Specification
+description: Describes the role of the chain specification in a network, how to specify the chain specification to use when starting a node, and how to customize and distribute chain specifications.
```

Comment:
The description is too long. It has 185 characters. It should be between 120 and 160